### ISSUE TITLE
Use RecipesBase instead of Plots

### ISSUE THREAD

**Issue by dlfivefifty** [RalphAS/Pseudospectra.jl#2]
It's better to create a recipe than a function that calls plots directly for a number of reasons, one of which that you don't need to load Plots to use a package.

**Comment by RalphAS** [comment]
I agree in principle, but so far I have been unable to write recipes that plot line art on top of contour plots, whereas that is easy with top-level Plots methods.
